test/fuzzer/duckfuzz/list_zip_no_args.test

```diff
@@ -0,0 +1,29 @@
+# name: test/fuzzer/duckfuzz/list_zip_no_args.test
+# description: Test List Zip with no arguments. 
+# group: [duckfuzz]
+
+require tpch
+
+statement ok
+call dbgen(sf=0.1);
```

Comment:
Can we remove this from the test? `dbgen` takes a reasonable amount of time to run and it doesn't really add anything to this test